apps/api/src/database/migrations/1732200721350-HotfixArchiveTable.ts

```diff
@@ -0,0 +1,37 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class HotfixArchiveTable1732200721350 implements MigrationInterface {
```

Comment:
This is incorrect.
The name of migration should mention what its actually doing(like add table, update table, alter table/column/index etc)...HotFix keyword is used for commits not code.

Reply:
Also once make sure in workbench/dbeaver columns are consistent between the two tables.

Reply:
> The name of migration should mention what its actually doing
Fixed 9647de9d55e8cbffd271ca954ff4ae9e78a11299

> Also once make sure in workbench/dbeaver columns are consistent between the two tables.
Have added the screenshots, kindly check that once please